Desktop/property-pro-app/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./footer.css";
+
+export default function Footer() {
+  return (
+    <div className="footer">
+      <div className="footer-first-child">
+        <span className="ab">ABOUT US</span>
```

Comment:
You may want to change your class namings to be more descriptive. "ab" is not a good class name you may use "about-us", also look out for other places to change the class names to be more descriptive.